src/pipelines.py

```diff
@@ -42,6 +42,50 @@ def unet(config, train_mode):
     return output
 
 
+def unet_tta(config):
+    save_output = False
+    load_saved_output = False
+    preprocessing, tta_generator = preprocessing_inference_tta(config, model_name='unet')
+
+    unet = Step(name='unet',
+                transformer=PyTorchUNet(**config.model['unet']),
+                input_data=['callback_input'],
+                input_steps=[preprocessing],
+                is_trainable=True,
+                cache_dirpath=config.env.cache_dirpath,
+                save_output=save_output, load_saved_output=load_saved_output)
+
+    tta_aggregator = Step(name='tta_aggregator',
```

Comment:
@taraspiotr I think we can extract the tta components to a seperate function. By doing so we could reuse them with double_unet